ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDataScienceProject/Storages.resource

```diff
@@ -54,10 +54,11 @@ Storage Size Should Be
     IF    "${pvc_status_phase}" == "Pending"
         # PVC hasn't been bound yet - only simple text with max storage should be shown
         ${storage_size_el}=    Set Variable
-        ...    ${STORAGE_SECTION_XP}//tr[@class="pf-c-table__expandable-row pf-m-expanded"]/td/div[strong[text()="Size"]]/div    # robocop: disable
+        ...    ${STORAGE_SECTION_XP}//td[contains(@data-label,"Size")]//div[strong[text()="Size"]]/div    # robocop: disable
         Wait Until Page Contains Element    ${storage_size_el}    timeout=20s
-        ${displayed_size}=    Get Text    ${storage_size_el}
-        Run Keyword And Continue On Failure    Should Be Equal As Strings    ${displayed_size}    Max ${size}Gi
+        ${displayed_sizes}=    Get All Text Under Element    ${storage_size_el}
+        ${matched_sizes}=    Get All Strings That Contain    ${displayed_sizes}    ${size}Gi
```

Comment:
why are you removing "Max"? when the PVC is not bounded to a workload it shows Max ${size}Gi, at least it used to do so last time I checked

Reply:
that should be handled by the IF/ELSE already present. iirc @jstourac worked on it. 

Max appears when the PVC is not bounded

Reply:
I restored the Max check (it required xpath fix)